style.css

```diff
@@ -0,0 +1,467 @@
+header {
+  width: 100%;
+  height: 60px;
+  padding: 0 16px;
+  background-color: rgb(245, 235, 235);
+  position: fixed;
+  top: 0;
+}
+
+nav {
+  display: flex;
+  height: 100%;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.start-pt {
+  display: flex;
+  height: 100%;
+  align-items: center;
+}
+
+.bars-info {
+  margin-right: 16px;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.nav-brand {
+  width: 80px;
+}
+
+.center {
+  display: flex;
+  width: 40%;
+  height: 100%;
+  align-items: center;
+}
+
+.search-bar {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  border: 1px solid;
+  border-color: lightgray;
+}
+
+.search-bar input[type=text] {
+  flex: 1;
+  padding: 4px 6px;
+  font-size: 16px;
+  font-family: "Roboto", "Noto", sans-serif;
+  font-weight: 400;
+  line-height: 24px;
+  border: none;
+  border-right: 1px solid;
+  border-color: lightgray;
+}
+
+.search-bar button {
+  padding: 4px 8px;
+  background: #f8f8f8;
+  border: none;
+  cursor: pointer;
+}
+
+.nav-right {
+  display: flex;
+  align-items: center;
+  margin-right: 32px;
+}
+
+.nav-right button {
+  margin-right: 8px;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.nav-ico {
+  width: 32px;
+  height: 32px;
+}
+
+.nav-right a img {
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+}
+
+main {
+  margin-top: 66px;
+  display: flex;
+  justify-content: center;
+  background-color: #f9f9f9;
+}
+
+.video-content {
+  width: 30%;
+}
+
+.video-player {
+  position: relative;
+  padding-bottom: 45%;
+  margin-top: 30px;
+  padding-top: 25px;
+  margin-right: 30px;
+  margin-left: 80px;
+  width: 90%;
+}
+
+.video-player iframe {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 95%;
+  height: 100%;
+}
+
+.vid-info {
+  padding: 20px 0 10px 0;
+  border-bottom: 1px solid;
+  border-color: lightgray;
+  margin-left: 80px;
+  font-size: 12px;
+}
+
+.btn-thumbs {
+  position: relative;
+  font-size: 10px;
+  left: 465px;
+  bottom: 30px;
+  margin-right: 180px;
+  cursor: pointer;
+}
+
```

Comment:
Please add the right widths and margins to your design and try to stick as much as possible to the youtube page
 design.
![like btn](https://user-images.githubusercontent.com/97598663/159270126-7c5d41f8-5949-4a64-87d1-1f4c0722307b.jpg)